sunpy/net/helioviewer.py

```diff
@@ -171,13 +184,19 @@ def download_jp2(self, date, observatory=None, instrument=None, detector=None,
         >>> hv = helioviewer.HelioviewerClient()  # doctest: +REMOTE_DATA
         >>> filepath = hv.download_jp2('2012/07/03 14:30:00', observatory='SDO',
         ...                            instrument='HMI', detector=None, measurement='continuum')  # doctest: +REMOTE_DATA
+        >>> filepath1 = hv.download_jp2('2012/07/03 14:30:00', observatory='SDO', measurement='continuum')  # doctest: +REMOTE_DATA
         >>> aia = sunpy.map.Map(filepath)  # doctest: +REMOTE_DATA
         >>> aia.peek()  # doctest: +SKIP
         """
+        data_dictionary = self.data_sources
```

Comment:
The same as above.

Reply:
Can I initialise the dictionary inside the init function? Then we can use it anywhere we need.

Reply:
We already have tho, its called `self.data_sources`

Reply:
Oh, looks like I got confused with something but now it's clear. Thanks